zeek/plugin/src/plugin.cc

```diff
@@ -120,6 +136,12 @@ const spicy::rt::Parser* plugin::Zeek_Spicy::Plugin::parserForFileAnalyzer(const
     return _file_analyzers_by_subtype[tag.Subtype()].parser;
 }
 
+#ifdef HAVE_PACKET_ANALYZERS
+const spicy::rt::Parser* plugin::Zeek_Spicy::Plugin::parserForPacketAnalyzer(const ::zeek::packet_analysis::Tag& tag) {
+    return _packet_analyzers_by_subtype[tag.Subtype()].parser;
```

Comment:
Like for the other analyzer stores we don't check or assert that the element `tag` is actually present in the vector. We could just return a `nullptr` for such cases.